vllm/v1/core/sched/interface.py

```diff
@@ -4,6 +4,11 @@
 from collections.abc import Iterable
 from typing import TYPE_CHECKING, Optional
 
+from vllm.config import VllmConfig
+from vllm.config.vllm import KVCacheConfig
+from vllm.multimodal import MULTIMODAL_REGISTRY, MultiModalRegistry
+from vllm.v1.structured_output import StructuredOutputManager
```

Comment:
These could all be moved into `TYPE_CHECKING` so they don't have to be imported at runtime